Kitodo-DataFormat/src/main/java/org/kitodo/dataformat/access/MetadataEntry.java

```diff
@@ -0,0 +1,87 @@
+/*
+ * (c) Kitodo. Key to digital objects e. V. <contact@kitodo.org>
+ *
+ * This file is part of the Kitodo project.
+ *
+ * It is licensed under GNU General Public License version 3 or later.
+ *
+ * For the full copyright and license information, please read the
+ * GPL3-License.txt file that was distributed with this source code.
+ */
+
+package org.kitodo.dataformat.access;
+
+import org.kitodo.api.dataformat.mets.MdSec;
+import org.kitodo.api.dataformat.mets.MetadataXmlElementAccessInterface;
+import org.kitodo.dataformat.metskitodo.MetadataType;
+
+/**
+ * A meta-data entry. This is probably the most important Java class of all
```

Comment:
Please get rid of the prose! Cut it down to what is important for development.
```
A meta-data entry consists of a key and a value. The key is stored in the superclass {@link Metadata}.
````

should be enough

Reply:
Okay, that can be shortened. TO DO